react/yarn.lock

```diff
@@ -3197,7 +3197,7 @@
     error-stack-parser "^2.0.6"
     find-up "^5.0.0"
     html-entities "^2.1.0"
-    loader-utils "^2.0.4"
+    loader-utils "^2.0.0"
```

Comment:
This must stay at `^2.0.4` because older versions are not secure. We had an alert [here](https://github.com/boxwise/boxtribute/security/dependabot/52) and @vahidbazzaz fixed it in #570

Reply:
@vahidbazzaz I have a problem here. when I run yarn install it downgrades the loader-utils subdependencies. I do not know why. I believe it is due to different versions of your local yarn and the yarn in the docker container (which I use). Could you run yarn install and check if it downgrades loader-utils on your end too?